### ISSUE TITLE
not able to connect via winrm

### ISSUE THREAD

**Issue by dushyant28** [test-kitchen/kitchen-ec2#175]
I am using kitchen-ec2 for windows2012r2 

My kitchen.yml looks like:

---
driver:
  name: ec2
  region: us-east-1
  availability_zone: d
  subnet_id: subnet-38d7b54f
  aws_ssh_key_id: test
  require_chef_omnibus: true
  instance_type: t2.micro
  associate_public_ip: true
  interface: dns
  security_group_ids: ["sg-c54e86a2"]

provisioner:
  name: chef_solo

platforms:
  - name: windows-2012r2
    driver:
      image_id: ami-5b9e6b30
    transport:
      username: test
      password: password

suites:
  - name: default
    run_list:
      - recipe[test::default]
    attributes:

>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: Failed to complete #create action: [A connection attempt failed because the connected party did not prop
erly respond after a period of time, or established connection failed because connected host has failed to respond. - co
nnect(2) for "ec2-52-3-107-64.compute-1.amazonaws.com" port 5985 (http://ec2-52-3-107-64.compute-1.amazonaws.com:5985)]

The ami that i am using is the base windows 2012 r2 ami. I am also able to telnet 5985 port from my workstation. Please help !

**Comment by dushyant28** [comment]
any updates on this ?

**Comment by zl4bv** [comment]
Hi @dushyant28, since you can access the instance could you run `winrm get winrm/config` and paste the output here?

**Comment by akemner** [comment]
@zl4bv same issue here as well.

```
PS C:\Users\Administrator> winrm get winrm/config
Config
    MaxEnvelopeSizekb = 500
    MaxTimeoutms = 1800000
    MaxBatchItems = 32000
    MaxProviderRequests = 4294967295
    Client
        NetworkDelayms = 5000
        URLPrefix = wsman
        AllowUnencrypted = false
        Auth
            Basic = true
            Digest = true
            Kerberos = true
            Negotiate = true
            Certificate = true
            CredSSP = false
        DefaultPorts
            HTTP = 5985
            HTTPS = 5986
        TrustedHosts = *
    Service
        RootSDDL = O:NSG:BAD:P(A;;GA;;;BA)(A;;GR;;;IU)S:P(AU;FA;GA;;;WD)(AU;SA;GXGW;;;WD)
        MaxConcurrentOperations = 4294967295
        MaxConcurrentOperationsPerUser = 1500
        EnumerationTimeoutms = 240000
        MaxConnections = 300
        MaxPacketRetrievalTimeSeconds = 120
        AllowUnencrypted = true
        Auth
            Basic = true
            Kerberos = true
            Negotiate = true
            Certificate = false
            CredSSP = false
            CbtHardeningLevel = Relaxed
        DefaultPorts
            HTTP = 5985
            HTTPS = 5986
        IPv4Filter = *
        IPv6Filter = *
        EnableCompatibilityHttpListener = false
        EnableCompatibilityHttpsListener = false
        CertificateThumbprint
        AllowRemoteAccess = true
    Winrs
        AllowRemoteShellAccess = true
        IdleTimeout = 7200000
        MaxConcurrentUsers = 10
        MaxShellRunTime = 2147483647
        MaxProcessesPerShell = 25
        MaxMemoryPerShellMB = 1024
        MaxShellsPerUser = 50
```

**Comment by akemner** [comment]
interestingly enough, if i run a kitchen create it will fail to connect. if i just wait and run kitchen converge a few mins later it connects and runs fine.

```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: Failed to complete #create action: [A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. - connect(2) for "10.84.17.133" port 5985 (http://10.84.17.133:5985)]
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration


kitchen converge windows
-----> Starting Kitchen (v1.4.2)
-----> Creating <windows-windows-2012r2>...
       Finished creating <windows-windows-2012r2> (0m0.00s).
-----> Converging <windows-windows-2012r2>...
       Preparing files for transfer
       Preparing dna.json
       Resolving cookbook dependencies with Berkshelf 3.3.0...
       Removing non-cookbook files before transfer
       Preparing validation.pem
       Preparing client.rb
-----> Installing Chef Omnibus (install only if missing)
```

**Comment by Stift** [comment]
@dushyant28 
Could you check this issue again as a proposed fix was just merged! #835

**Comment by cheeseplus** [comment]
Closing this as fixed via above.
